### ISSUE TITLE
ADSBExchange Error

### ISSUE THREAD

**Issue by HunterLewis000** [nycmadu/VRCLiveTraffic#1]
Every few lines reads a warning that says, "Error fetching index 6232 from ADSBExchange: Status code 403 Forbidden indicates failure".

By the way, thank you so much for getting the old project working again. I thought it would never be fixed.

**Comment by nycmadu** [comment]
ADS-B Exchange has updated its API access policies. As of April 25, 2019, the previously open public API was discontinued, and access now requires an API key.

I would try using something airplanes.live API. I could look further into updating that.

**Comment by HunterLewis000** [comment]
Okay, makes sense. If individually I bought an API key would I be able to get this to work? Or is something like this not even worth it? Also, in terms of good radar scopes for ADSB data, is integrating it into VRC still the best way to go? I saw something about DG Scope but had no clue how to compile it or use it.

Thanks

**Comment by nycmadu** [comment]
Yes, but it would need some changes with mods.rs. I don't think buying the API key is worth it. Airplanes.live api is public (which does the same thing as ADS-B Exchange.) Going forward I don't think it's even worth using VRC live traffic. Dstars (or DG Scope) using VRC and [CRC](https://vnas.vatsim.net/crc) built into Dstars. You could build it on Visual Solutions (The way I compiled it) or via dotnet dev.

If you need any help with DG Scope, I could try my best over text to help or I can help you live on discord. (c4blitzem)
